### DISCUSSION TITLE
Adding a dialogue box before exiting editor without saving

### DISCUSSION THREAD

**Discussion by Paradox304** [SmartlyDressedGames/Unturned-3.x-Community#3987] [Ideas]
This is just a quality of life change, but if it's possible can we get a dialogue box saying "You have unsaved changes, are you sure you want to exit without saving them" and have an option to Save or Exit without saving. I can't imagine how many times map developers have exited out of their editor without saving and realize hours of editing has gone down the drain.

**Comment by Jdance-Media** [comment]
Bump.
